.flake8

```diff
@@ -4,6 +4,8 @@ max-line-length = 119
 ignore = W503
 
 exclude =
+    docs/conf.py
```

Comment:
For future reference, I would not mind _not_ excluding this file, and instead clean it up. All that commented out code and comments are not needed when you have documentation.